### DISCUSSION TITLE
一些设计稿（Figma）

### DISCUSSION THREAD

**Discussion by lynzrand** [chafan-dev/chafan-frontend#29] [Ideas]
![图片](https://user-images.githubusercontent.com/10259119/110199753-e3f9f180-7e94-11eb-9527-07d3e8d83568.png)

![图片](https://user-images.githubusercontent.com/10259119/110199756-e9573c00-7e94-11eb-96e6-b7f3dffcd354.png)

见 https://www.figma.com/file/zZNEleFG6rkyIJddHDN8PJ/Cha.Fan

如果有编辑需求请在 Discord 群戳咱（`Rynco Maekawa#0379`）

**Comment by izgzhen** [comment]
Related https://github.com/chafan-dev/chafan-frontend/issues/30

**Comment by lynzrand** [comment]
![图片](https://user-images.githubusercontent.com/10259119/110211168-563bf780-7ed0-11eb-9c0d-a035f8a8cdc7.png)
![图片](https://user-images.githubusercontent.com/10259119/110211175-5936e800-7ed0-11eb-8829-5929c62fd1c8.png)
![图片](https://user-images.githubusercontent.com/10259119/110211180-5f2cc900-7ed0-11eb-8096-33773c44f5de.png)
